packages/orm/mikro-orm/test/helpers/services/UserService.ts

```diff
@@ -1,6 +1,7 @@
 import {Injectable} from "@tsed/di";
 import {Connection} from "../../../src";
-import {MikroORM} from "@mikro-orm/core";
+import {EntityManager, MikroORM} from "@mikro-orm/core";
+import {Em} from "../../../src/decorators/entityManager";
```

Comment:
Use shorthand imports here. Also, it lets us optimize imports merging 2th and 4th lines 

```suggestion
import {Em} from "../../../src";
```

Reply:
done